tutorials/tmva/TMVACrossValidationRegression.C

```diff
@@ -7,13 +7,13 @@
 /// As input data is used a toy-MC sample consisting of two gaussian
 /// distributions.
 ///
-/// The output file "TMVA.root" can be analysed with the use of dedicated
+/// The output file "TMVACV.root" can be analysed with the use of dedicated
```

Comment:
Here the output file name  should be "TMVARegCv.root"

Reply:
OK, no problem, thanks for letting me know!